games/minimal/mods/unittests/random.lua

```diff
@@ -0,0 +1,10 @@
+function unittests.test_random()
+	-- Try out PseudoRandom
+	minetest.log("action", "[unittests] Testing PseudoRandom ...")
+	local pseudo = PseudoRandom(13)
```

Comment:
Should work differently; PseudoRandom is not guaranteed to always return those numbers, it should only return the same numbers each run, depending on the seed. So I'd suggest going with something like:
```lua
for i = 1, 13 do
    local pseudo = PseudoRandom(i)
    local pseudo2 = PseudoRandom(i)
    for j = 1, 10 do
         assert(pseudo:next() == pseudo2:next())
    end
end
```
But we can probably drop this test altogether.

Reply:
This is actually guaranteed.